packages/common/src/lib/tool/shared/toolbox.ts

```diff
@@ -109,6 +117,40 @@ export class Toolbox {
     this.activateTool(previous);
   }
 
+  /**
+   * Activate the tool below, if any
+   */
+  activateBelowTool() {
+    const arrayTools = this.getToolbar();
+    const index = arrayTools.findIndex(t => t === this.activeTool$.getValue().name);
+    if (arrayTools[index + 1] !== undefined) {
+      this.deactivateTool();
+      const below = arrayTools[index + 1];
+      this.activateTool(below);
+    } else {
```

Comment:
If the active tool is the last one, this will activate the first tool right? I think it's a bit odd to do that. Also, will the scrolling work properly?

Reply:
Yes, it will activate the first one. These functions were made before we took the decision to put scroll button. So, right now, these aren't used. I kept them here in case maybe someone has a great idea in the future.

Reply:
I see. In that case, I think we should remove them.